eslint-bridge/tests/tools/testers/comment-based/launcher.test.ts

```
@@ -46,26 +46,29 @@ import { decorateExternalRules } from 'linting/eslint/linter/decoration';
 
 const fixtures = path.join(__dirname, '../../../linting/eslint/rules/comment-based');
 
+function extractRuleOptions(testFiles, rule) {
+  if (testFiles.includes(`${rule}.json`)) {
+    try {
+      return JSON.parse(fs.readFileSync(path.join(fixtures, `${rule}.json`), { encoding: 'utf8' }));
+    } catch {}
```

I think we should either propagate the exception or not catch it in case the rule options are malformed for some reason. Here, we are just making it quiet.